src/style.css

```diff
@@ -62,6 +62,9 @@
   --color-heading: #1a202c;
   --color-heading-black: black;
   --color-accent: #d1dce5;
+  --color-white: #f7fafc;
+  --letter-spacing-2: 2px;
```

Comment:
I think this should be `2.5px` instead. Damn those designers and their inconsistent styles!